docs/porting/connectivity/MeshMAC.md

```diff
@@ -1,6 +1,9 @@
 <h1 id="mac-port">MAC API porting</h1>
 
 Nanostack has a lower level API for the IEEE 802.15.4-2006 MAC standard. This enables developers to support different MACs, be it SW or HW based solutions. Nanostack offers SW MAC, which you can use when your board does not have 15.4 MAC available.
+SW MAC also supports subset of IEEE 802.15.4-2015 MAC standard. Supported features are:
+- IEEE 802.15.4-2015 enhanced ack
+- Header and paylod Information elements set by MAC user. 
 
 ## SW MAC
```

Comment:
@AnotherButler Is this SW MAC chapter then also at wrong file. Reason Why I put MAC updates there was that.